ports/libjuice/vcpkg.json

```
@@ -1,8 +1,9 @@
 {
   "name": "libjuice",
-  "version": "0.9.8",
+  "version": "1.0.0",
   "description": "The library is a simplified implementation of the Interactive Connectivity Establishment (ICE) protocol in C for POSIX platforms (including Linux and Apple macOS) and Microsoft Windows.",
   "homepage": "https://github.com/paullouisageneau/libjuice",
+  "license": "LGPL-2.1",
```

```suggestion
  "license": "LGPL-2.1",
```
`LGPL-2.1`- only or later?